src/components/LandingPage.js

```diff
@@ -15,8 +15,6 @@ function LandingPage(props) {
     };
   }, []);
 
-  console.log(props);
```

Comment:
Is this a debug print or part of the app? If debug, it should be removed.

Reply:
I think this is a debug print. Personally, I treat all unformatted logs as debug prints.